From 360351b9b99bba0a278923c4237d158cbd726df5 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sun, 12 Feb 2012 23:08:42 -0500
Subject: [PATCH 1/3] nscd: add missing newline to warning msg

Trying to run `passwd` without nscd running shows the messages:
...
Re-enter new password:
passwd: nscd exited with status 1passwd: Failed to flush the nscd cache.
passwd: nscd exited with status 1passwd: Failed to flush the nscd cache.
passwd: nscd exited with status 1passwd: Failed to flush the nscd cache.
passwd: nscd exited with status 1passwd: Failed to flush the nscd cache.
passwd: password changed.
...

You can see the status message missing a newline, so add it.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 lib/nscd.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/nscd.c b/lib/nscd.c
index 227c205..ea1dc55 100644
--- a/lib/nscd.c
+++ b/lib/nscd.c
@@ -40,7 +40,7 @@ int nscd_flush_cache (const char *service)
 		   interpreter that is missing.  Probably the former. */
 		return 0;
 	} else if (code != 0) {
-		(void) fprintf (stderr, _("%s: nscd exited with status %d"),
+		(void) fprintf (stderr, _("%s: nscd exited with status %d\n"),
 		                Prog, code);
 		(void) fprintf (stderr, _(MSG_NSCD_FLUSH_CACHE_FAILED), Prog);
 		return -1;
-- 
1.7.8.4

